floodviz/static/js/peakflow.js

```diff
@@ -0,0 +1,94 @@
+document.addEventListener("DOMContentLoaded", function(event) { 
+
+	"use strict";
+
+	// Collect and set peakflow bar chart aspect ratio data
+	var peakflow_bar = document.getElementById('peakflow_bar');
+	peakflow_bar.style.height = FV.peakmeta['height'];
+	peakflow_bar.style.width = FV.peakmeta['width'];
+
+	var margin = {bottom: 100, right: 120, left: 120, top: 150};
+	var width = parseInt(FV.peakmeta['width']);
+	var	height = parseInt(FV.peakmeta['height']);
```

Comment:
Not sure why this is happening here. My text editor isn't showing this even after format indentation shortkey